cmd/nodecmd/create_aws.go

```diff
@@ -523,7 +523,9 @@ func createAWSInstances(
 		if len(failedNodes) > 0 {
 			ux.Logger.PrintToUser("Failed nodes: ")
 			for node, err := range failedNodes {
-				ux.Logger.PrintToUser(fmt.Sprintf("Failed to destroy node %s due to %s", node, err))
+				if enableLogs {
```

Comment:
i feel like we should still display logs so it will be up to the user to still investigate.